src/AddressRegistry.Api.Oslo/Infrastructure/Elastic/AddressApiStreetNameElasticsearchClient-SearchStreetNames.cs

```diff
@@ -35,107 +29,57 @@ public async Task<StreetNameSearchResult> SearchStreetNames(
                         .Query(q =>
                             q.Bool(x =>
                             {
-                                if(status is not null)
-                                    x.Filter(f => f.Term(t => t.Field(ToCamelCase(nameof(StreetNameSearchDocument.Status))!).Value(status.ToString())));
-                                else
-                                    x.Filter(f => f.Term(t => t.Field(ToCamelCase(nameof(StreetNameSearchDocument.Active))!).Value(true)));
-
-                                var mustQueries = new List<Action<QueryDescriptor<StreetNameSearchDocument>>>();
-                                var shouldMunicipalityOrPostalQueries = new List<Action<QueryDescriptor<StreetNameSearchDocument>>>();
-
-                                mustQueries.Add(q2 =>
-                                    q2.Bool(b =>
-                                    {
-                                        b.Must(must =>
-                                            must.Nested(full =>
-                                                full
-                                                    .Path(_fullStreetNames)
-                                                    .Query(fullAddressQuery =>
-                                                    {
-                                                        fullAddressQuery.Bool(b => b.Should(s =>
-                                                        {
-                                                            if (!query.Contains(' '))
-                                                            {
-                                                                s.Prefix(p =>
-                                                                    p
-                                                                        .Field($"{_fullStreetNames}.{NameSpelling}")
-                                                                        .Value(query)
-                                                                        .Boost(3));
-                                                            }
-
-                                                            s.MatchPhrase(mp =>
-                                                                mp
-                                                                    .Field($"{_fullStreetNames}.{NameSpelling}")
-                                                                    .Query(query)
-                                                                    .Slop(10));
-                                                        }));
-                                                    })
-                                                    .InnerHits(c =>
-                                                        c.Size(1))
-                                            ));
-                                    }));
+                                var conditions = new List<Action<QueryDescriptor<StreetNameSearchDocument>>>();
```

Comment:
filter doesn't impact scoring and is more performant, since active/status and niscode don't impact score filter is better.